pkg/Microsoft.ML.HalLearners/Microsoft.ML.HalLearners.nupkgproj

```diff
@@ -7,7 +7,6 @@
 
   <ItemGroup>
     <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
-    <PackageReference Include="MlNetMklDeps" Version="$(MlNetMklDepsPackageVersion)" />
```

Comment:
> <PackageReference Include="MlNetMklDeps" Version="$(MlNetMklDepsPackageVersion)" / [](start = 3, length = 83)

how will the Hal Learners package suggest that the MlNetMklDeps NuGet needs to be installed as well?

Reply:
@sfilipi - with this change we are no longer going to need to publish the MlNetMklDeps package to nuget.org.  We will no longer have a public dependency from `Microsoft.ML.HalLearners` to `MlNetMklDeps`.  Instead, we are taking the assemblies out of the `MlNetMklDeps` package and packaging them inside `Microsoft.ML.HalLearners`.

Reply:
if we're packaging the binaries, we'd need the intel notice in there too.

Reply:
oooh, good point.

@codemzs - can you add this as well?  We won't be able to ship without it.  Either this PR or another one.  But we will need this if we are going to ship HalLearners in `0.4`

Reply:
Done, thanks @sfilipi and @eerhardt